datalad/interface/ls.py

```diff
@@ -189,6 +201,88 @@ def annex_local_size(self):
         info = self.info
         return info['local annex size'] if info else None
 
+
+@auto_repr
+class FsModel(DsModel):
+
+    __slots__ = ['_path', '_info', 'repo']
+
+    def __init__(self, path, repo=None):
+        self._path = path  # fs path to the node, can be overridden
+        self._info = None
+        self.repo = DsModel(Dataset(path)) if self.type_ == 'annex' else repo  # parent repository associated with node
+        self._branch = None
+
+    @property
+    def symlink(self):
+        """if symlink returns path the symlink points to else returns False"""
```

Comment:
IMHO better to return `None`  (yet to see how it is used).  It would eval to False if asked as 'bool', but also would have clearer semantic I think